src/javascripts/ng-admin/Crud/button/maBackButton.js

```diff
@@ -7,16 +7,19 @@ define(function () {
         return {
             restrict: 'E',
             scope: {
-                'size': '@'
+                'size': '@',
```

Comment:
no need for quotes for the object keys. Same for every other directive in the PR